src/handlers/debugHandler.ts

```diff
@@ -9,29 +9,37 @@ import { Settings } from "../Settings";
 import { executeInTerminal } from "../utils/mavenUtils";
 
 export async function debugHandler(goal: PluginGoal): Promise<void> {
-    const pomfile: string = goal.plugin.project.pomPath;
-    const command: string = goal.name;
-    await debugCommand({ command, pomfile });
+    await debugCommand({
+        command: goal.name,
+        pomfile: goal.plugin.project.pomPath,
+        projectName: goal.plugin.project.name

```

Comment:
For a maven project, what does debugger extension take for `projectName`. Is it `project.artifactId` or `project.name`? @testforstephen Can you confirm?

Reply:
It uses `artifactId` as projectName.